gammapy/analysis/core.py

```diff
@@ -79,33 +78,36 @@ def get_observations(self):
             raise FileNotFoundError(f"Datastore not found: {path}")
 
         log.info("Fetching observations.")
-        data_settings = self.config.observations
-        selected_obs = ObservationTable()
-        obs_list = self.datastore.get_observations()
-        if len(self.config.observations.obs_ids):
-            obs_list = self.datastore.get_observations(data_settings.obs_ids)
-        selected_obs["OBS_ID"] = [obs.obs_id for obs in obs_list]
-        ids = selected_obs["OBS_ID"].tolist()
-        # TODO
-        # if self.config.observations.obs_file:
-        # add obs_ids from file
-        # ids.extend(selected_obs["OBS_ID"].tolist())
-        if data_settings.obs_cone.lon is not None:
-            # TODO remove border keyword
+        observations_settings = self.config.observations
+        if len(observations_settings.obs_ids) and observations_settings.obs_file is not None:
+            raise ValueError("Values for both parameters obs_ids and obs_file are not accepted.")
+        elif not len(observations_settings.obs_ids) and observations_settings.obs_file is None:
+            obs_list = self.datastore.get_observations()
+            ids = [obs.obs_id for obs in obs_list]
+        elif len(observations_settings.obs_ids):
+            obs_list = self.datastore.get_observations(observations_settings.obs_ids)
+            ids = [obs.obs_id for obs in obs_list]
+        else:
+            path = make_path(self.config.observations.obs_file)
+            with open(path, 'r') as f:
+                ids = [int(obs_id) for obs_id in f.readlines()]
```

Comment:
Even if it's a very simple format here, I would suggest to use `Table.read(path, format="ascii.csv")` or some other ascii variant format here, just for consistency with the rest of Gammapy where we always used `Table.read` to read any table.